### DISCUSSION TITLE
rerouting the admin panel from a vps

### DISCUSSION THREAD

**Discussion by posaidon-09** [pocketbase/pocketbase#2500] [Q&A] [answered]
i was recently given ssh access to a vps owned by my friend and we're planning to host a website with it for our discord bot
i would like to know how would i be able to use the admin panel on my own network

**Comment by ganigeorgiev** [comment, marked as answer]
I'm not sure that I understand the question and what do you mean by _rerouting_.

You can specify the address to start the server with the `--http` flag (_the `--https` flag and auto Let's encrypt requires valid public domain name_).

```sh
./pocketbase serve --http="YOUR_IP:8090"
```

If you haven't already, for a minimal deployment setup you can check https://pocketbase.io/docs/going-to-production/#minimal-setup.

If by _rerouting_ you mean reverse proxy, then you can find a nginx example in https://pocketbase.io/docs/going-to-production/#using-reverse-proxy (_for [apache is similar](https://httpd.apache.org/docs/2.4/howto/reverse_proxy.html)_).
